buildozer/targets/android.py

```diff
@@ -798,9 +798,22 @@ def build_package(self):
             pass
 
         # XXX found how the apk name is really built from the title
+        #gradle_files = ["build.gradle", "gradle", "gradlew"]
+        #is_gradle_build = any((
+        #    exists(join(dist_dir, x)) for x in gradle_files))
+        #if is_gradle_build:
+	__sdk_dir = self.android_sdk_dir
+        build_tools_versions = os.listdir(join(__sdk_dir, 'build-tools'))
+        build_tools_versions = sorted(build_tools_versions, key=LooseVersion)
+        build_tools_version = build_tools_versions[-1]
         gradle_files = ["build.gradle", "gradle", "gradlew"]
-        is_gradle_build = any((
-            exists(join(dist_dir, x)) for x in gradle_files))
+        is_gradle_build = any((exists(join(dist_dir, x)) for x in gradle_files)) and build_tools_version >= '25.0'
+
+
+        
+        # gradle_files = ["build.gradle", "gradle", "gradlew"]
```

Comment:
Sorry, guys.. this is a total newb questions, but I can't edit the android.py... What's the procedure here?